src/types/type_row.rs

```diff
@@ -11,9 +11,12 @@ use super::Type;
 use crate::utils::display_list;
 use delegate::delegate;
 
+#[cfg(feature = "pyo3")]
+use pyo3::prelude::*;
```

Comment:
Why are we always importing all of the pyo3 prelude when we seem to just be using `pyclass`?